scripts/DeployTEEVerifier.s.sol

```
@@ -0,0 +1,38 @@
+pragma solidity 0.8.25;
+
+import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {CertManager} from "@nitro-validator/CertManager.sol";
+import {EspressoTEEVerifier} from "src/EspressoTEEVerifier.sol";
+import {IEspressoSGXTEEVerifier} from "../src/interface/IEspressoSGXTEEVerifier.sol";
+import {IEspressoNitroTEEVerifier} from "../src/interface/IEspressoNitroTEEVerifier.sol";
+
+contract DeployTEEVerifier is Script {
+
+    function run() external {
+        vm.startBroadcast();
+
+        address sgxVerifierAddr = vm.envAddress("SGX_VERIFIER_ADDRESS");
```

should we throw errors here is it is not set?

---

https://github.com/EspressoSystems/espresso-tee-contracts/pull/8/commits/c975bb4fb44ed9dcc92a5438c861eeb07dd1d7fc